cpp/src/tsne/barnes_hut_tsne.cuh

```diff
@@ -46,14 +47,15 @@ namespace TSNE {
  * @param[in] pre_momentum: The momentum used during the exaggeration phase.
  * @param[in] post_momentum: The momentum used after the exaggeration phase.
  * @param[in] random_state: Set this to -1 for pure random intializations or >= 0 for reproducible outputs.
- * @param[in] initialize_embeddings: Whether to overwrite the current Y vector with random noise.
  */
+
 template <typename value_idx, typename value_t>
 void Barnes_Hut(value_t *VAL, const value_idx *COL, const value_idx *ROW,
                 const value_idx NNZ, const raft::handle_t &handle, value_t *Y,
                 const value_idx n, const float theta = 0.5f,
                 const float epssq = 0.0025,
                 const float early_exaggeration = 12.0f,
+                const float late_exaggeration = 1.0f,
```

Comment:
This can be removed from the argument list until it's fully supported in barnes-hut